Resources/Textures/Objects/Specific/Hydroponics/crystal_thistle.rsi/meta.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
rsi should be in DeltaV folder, then update the yml

Reply:
Done and tested!